testing/tests/DevExpress.exporter/jspdfParts/jspdf_v2.dataGrid.tests.js

```diff
@@ -197,6 +203,336 @@ QUnit.module('Table', moduleConfig, () => {
+            columns: [{ caption: 'line1' }]
+        });
+
+        const expectedLog = [
+            'text,line1,10,45,{baseline:middle}',
+            'setLineWidth,1',
+            'rect,10,15,100,60'
+        ];
+
+        const onExporting = (e) => {
+            e.rowHeight = 60;
+        };
+
+        exportDataGrid(doc, dataGrid, { topLeft: { x: 10, y: 15 }, columnWidths: [ 100 ], onRowExporting: onExporting }).then(() => {
+            // doc.save();
+            assert.deepEqual(doc.__log, expectedLog);
+            done();
+        });
+    });
+
+    QUnit.test('1 col - 1 text line. fontSize 20, lineHeight: default', function(assert) {
+        const done = assert.async();
+        const doc = createMockPdfDoc();
+
+        const dataGrid = createDataGrid({
+            columns: [{ caption: 'line1' }]
+        });
+
+        const expectedLog = [
+            'setFontSize,20',
+            'text,line1,10,45,{baseline:middle}',
+            'setLineWidth,1',
+            'rect,10,15,100,60'
+        ];
+
+        const onExporting = (e) => {
+            e.rowHeight = 60;
+        };
+
+        doc.setFontSize(20);
+        exportDataGrid(doc, dataGrid, { topLeft: { x: 10, y: 15 }, columnWidths: [ 100 ], onRowExporting: onExporting }).then(() => {
+            // doc.save();
+            assert.deepEqual(doc.__log, expectedLog);
+            done();
+        });
+    });
+
+    QUnit.test('1 col - 1 text line. fontSize 20, lineHeight: 1', function(assert) {
+        const done = assert.async();
+        const doc = createMockPdfDoc();
+
+        const dataGrid = createDataGrid({
+            columns: [{ caption: 'line1' }]
+        });
+
+        const expectedLog = [
+            'setFontSize,20',
+            'setLineHeightFactor,1',
+            'text,line1,10,42,{baseline:middle}',
+            'setLineWidth,1',
+            'rect,10,15,100,60'
+        ];
+
+        const onExporting = (e) => {
+            e.rowHeight = 60;
+        };
+
+        doc.setFontSize(20);
+        doc.setLineHeightFactor(1);
+        exportDataGrid(doc, dataGrid, { topLeft: { x: 10, y: 15 }, columnWidths: [ 100 ], onRowExporting: onExporting }).then(() => {
+            // doc.save();
+            assert.deepEqual(doc.__log, expectedLog);
+            done();
+        });
+    });
+
+    QUnit.test('1 col - 1 text line. fontSize 20, lineHeight: 1.5', function(assert) {
+        const done = assert.async();
+        const doc = createMockPdfDoc();
+
+        const dataGrid = createDataGrid({
+            columns: [{ caption: 'line1' }]
+        });
+
+        const expectedLog = [
+            'setFontSize,20',
+            'setLineHeightFactor,1.5',
+            'text,line1,10,52,{baseline:middle}',
+            'setLineWidth,1',
+            'rect,10,15,100,60'
+        ];
+
+        const onExporting = (e) => {
+            e.rowHeight = 60;
+        };
+
+        doc.setFontSize(20);
+        doc.setLineHeightFactor(1.5);
```

Comment:
I think that these settings can be different for each cell, clients need a way to specify it for entire grid and for each cell